app/common/view.c

```diff
@@ -47,12 +47,14 @@ void h_approve(__Z_UNUSED unsigned int _) {
     }
 }
 
-void h_reject(__Z_UNUSED unsigned int _) {
+void h_reject(uint8_t requireReply) {
```

Comment:
I think the signature with `unsigned int` was required by Ledger's api. Not sure.
Maybe it is a good idea to check